enterprise/configuration-files/default.template.conf

```diff
@@ -0,0 +1,23 @@
+server {
```

Comment:
can we put nginx somewhere in the name of this file?  it is confusing because we have the arm conf's as well.